.gitignore

```diff
@@ -10,4 +10,3 @@ tags
 uploaded_tasks/
 uploaded_images/
 uploaded_files/
-.idea
```

Comment:
We have .idea in .gitignore currently, and I think we want it to stay. So we need a pull request without any .gitignore changes in it.